service/controller/cloudconfig/types.go

```diff
@@ -46,3 +49,10 @@ type certificateDecrypterUnitParams struct {
 type ingressLBFileParams struct {
 	ClusterDNSDomain string
 }
+
+type IgnitionTemplateData struct {
+	CustomObject providerv1alpha1.AzureConfig
```

Comment:
Maybe sort the keys in alphabetical order?

Reply:
I'm not actually sure what's the right thing here. Above types are non-exported and this one is exported. In `aws-operator` it was like this as well. I'd maybe prefer this instead of mixing exported and non-exported types?